packages/create-rise/src/cli.ts

```diff
@@ -122,10 +114,39 @@ async function copyAdditionalTemplateFiles(root: string) {
   await $`cp ${path.join(source, '../_gitignore')} ${path.join(root, '.gitignore')}`
 }
 
-createRise().catch((e) => {
+const opts = minimist<Options>(process.argv.slice(2), {
```

Comment:
Can add a `--non-interactive` flag, might helpful in CI.

https://docs.expo.dev/more/create-expo/#--yes

Reply:
feel free to send over in a separate PR!